lib/inc/drogon/WebSocketConnection.h

```diff
@@ -116,6 +117,26 @@ class WebSocketConnection
         std::string_view msg,
         const WebSocketMessageType type = WebSocketMessageType::Text) = 0;
 
+    /**
+     * @brief Send a message to the peer
+     *
+     * @param json The JSON message to be sent.
+     * @param type The message type.
+     */
+    virtual void send(
+        Json::Value &json,
+        const WebSocketMessageType type = WebSocketMessageType::Text) = 0;
+
+    /**
+     * @brief Send a message to the peer
+     *
+     * @param json The JSON message to be sent.
+     * @param type The message type.
+     */
+    virtual void send(
+        const Json::Value *json,
+        const WebSocketMessageType type = WebSocketMessageType::Text) = 0;
+
```

Comment:
I think adding only one method with const Json::Value & type is good enough.

Reply:
The first commit was only one overload for `const Json::Value&`, however, it didn't compile due to the implicit constructor of `Json::Value(const String&)`, and it became ambigious with WebSocket connection's `send(std::string_view)`

Reply:
Change the method name to sendJson() could resolve the problem.